recipes/demcmc/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "demcmc" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/demcmc-{{ version }}.tar.gz
+  sha256: e64eda8ad27dbf4f7f5b21f6607e7f8f4071d22e4e4c4bf0b6be1718259a8cb9
+
+build:
+  skip: true  # [py<38]
```

Since this is pure Python it'd be much better to make this noarch...
```suggestion
  noarch: python
```